client/src/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Header() {
+  return <HeaderStyled></HeaderStyled>
+}
+
+const HeaderStyled = styled.header`
+  height: 76px;
+  background: #fd474b;
```

Comment:
Wondering the same as above, let me know :)

Reply:
Had some CSS issues that could only be fixed when overlaying two header styles...duh still fixing :D